src/officehours_ui/static/css/main.css

```diff
@@ -467,6 +467,12 @@ h3.alert-heading {
 	border: 1px solid #cecbc9;
 }
 
+.form-p {
```

Comment:
It might be good to rename this something like `renaming-controls` to make it clear that's the group of elements it's intended for.